src/binfhe/include/binfhe-constants.h

```diff
@@ -111,8 +111,8 @@ std::ostream& operator<<(std::ostream& s, BINGATE f);
  * @brief Lists all modes for secret keys supported
  */
 enum SECRET_KEY_DIST {
-    GAUSSIAN        = 0,
-    UNIFORM_TERNARY = 1,
+    GAUSSIAN_BINFHE        = 0,
```

Comment:
Why use a different name? Better access them as SECRET_KEY_DISTR::GAUSSIAN

Reply:
Made a separate header file in core/lattice to be able to use the same SecretKeyDist enum for both pke and binfhe.